vllm/core/block_manager_v2.py

```diff
@@ -9,7 +9,20 @@
 from vllm.sequence import Sequence, SequenceGroup, SequenceStatus
 from vllm.utils import Device
 
+'''
+Exception strings for non-implemented encoder/decoder scenarios
+'''
+
+str_not_impl_enc_dec_swa = \
```

Comment:
same comment about uppercase for constants

Reply:
Thanks! Fixed in most recent push.